Dockerfile

```diff
@@ -1,5 +1,26 @@
-FROM openjdk:11-jre-slim
+# Start with a base image containing Java runtime
+#FROM openjdk:11.0.6-stretch
+#FROM openjdk:12-jdk-alpine
+FROM openjdk:12-jdk-oraclelinux7
+# Add Maintainer Info
+LABEL maintainer="gabriele.deluca@eng.it"
+
+#Install whois service
+RUN yum install -y whois
+
+# Add a volume pointing to /tmp
 VOLUME /tmp
-COPY target/ids-usage-control-app-0.0.1-SNAPSHOT.jar app.jar
-ENTRYPOINT ["java","-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
-EXPOSE 9552
\ No newline at end of file
+
+# Make port 9552 available to the world outside this container
+EXPOSE 9552
+
+# Add the application's jar to the container
+ADD target/true-connector-uc_data_app.jar true-connector-uc_data_app.jar
```

Comment:
Why ADD and not COPY?
https://phoenixnap.com/kb/docker-add-vs-copy
I'm not that expert of Docker, but Docker suggest that usage of copy is preferred way to go. Please also check some other resources if we should use copy and not add

Reply:
We use ADD everywhere that's why I didn't change it. But from reading that link and this one https://docs.docker.com/develop/develop-images/dockerfile_best-practices/, I agree that we should change it to COPY.